app/views/ReactionsView/index.js

```diff
@@ -0,0 +1,111 @@
+import React, { useEffect } from 'react';
+import {
+	View, Text, TouchableOpacity
+} from 'react-native';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { withSafeAreaInsets } from 'react-native-safe-area-context';
+import ScrollableTabView, { ScrollableTabBar } from 'react-native-scrollable-tab-view';
+
+import I18n from '../../i18n';
+import { withTheme } from '../../theme';
+import { themes } from '../../constants/colors';
+import { withDimensions } from '../../dimensions';
+import sharedStyles from './styles';
+import SafeAreaView from '../../containers/SafeAreaView';
+import Emoji from '../../containers/message/Emoji';
+
+import TabPage from './renderPage';
+
+const scrollProps = {
+	keyboardShouldPersistTaps: 'always',
+	keyboardDismissMode: 'none'
+};
+
+const RenderLabel = React.memo(({
```

Comment:
`render*` is a concept from `React.Class` and it should be used as functions only to be called as `{renderSomething()}`.
Creating a component (with capital letter) is anti-pattern.
It could be outside this file btw.

Reply:
Refactored to TabLabel